src/libs/OptionsListUtils.ts

```diff
@@ -1402,6 +1403,35 @@ function createOptionFromReport(report: Report, personalDetails: OnyxEntry<Perso
     };
 }
 
+/**
+ * Options need to be sorted in the specific order
+ * @param options - list of options to be sorted
+ * @param searchValue - search string
+ * @returns a sorted list of options
+ */
+function orderOptions(options: ReportUtils.OptionData[], searchValue: string | undefined) {
```

Comment:
let's add some unit tests to cover this function?

Reply:
tests for ordering are already covered with `filterOptions` tests